dashboard/README.md

```diff
@@ -4,53 +4,49 @@
 
 Pbench Dashboard is a web-based platform for consuming indexed performance benchmark data. The platform provides a consolidated view of benchmark data within tables, charts, and other powerful visualizations. Users are able to quickly navigate through benchmark data and tune analytics through comparison tools for in-depth analysis.
 
-  
+# Dashboard directory structure
 
-## Scaffolding
+## public
 
-  
+Contains the root application `index.html` and React template artifacts.
 
-```bash
- .
- ├── public									# HTML entry
- ├── server 								# endpoint configuration
- └── src
-     ├── actions							# redux actions
-     ├── assets
-     │   ├── constants						# constants used in code
-     │   ├── images							# images
-     │   └── logo							# Pbench logos
-     ├── modules
-     │   ├── components						# functional components
-     │   └── containers						# parent components
-     ├── reducers							# redux reducer
-     ├── store								# redux store
-     └── utils								# utility functions
-```
+(BTW, do we really want/need to retain the React logos here? We don't use them, do
+we?)
 
-  
+## server
 
-## Assets
+The source for an NPM express server that's used in local developer mode to reflect
+the server `/api/v1/endpoints` API call to a remote server. This enables local
+debugging against a real Pbench Server.
 
-  
+## src
+
+The Pbench Dashboard Javascript source plus additional CSS/LESS and artifacts.
+
+### assets
 
 Assets placed in the `src/assets/images` directory are only referenced within component or layout definitions and are packaged in the generated `***.js` file during the build process.
 
+### modules
+  
+### reducers
   
+### store
 
-## Prerequisites
+### utils
 
-  
+## Prerequisites
+ 
 
 ## Install Node JS
 
-Refer [https://nodejs.org/en/] (Node.js) official website
+Refer to the [Node.js](https://nodejs.org) official website
 
   
 
 ## Install create-react-app
 
-Install create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
+Install "create-react-app npm package globally. This will help to easily run the project and also build the source files easily. Use the following command to install create-react-app
```

Comment:
s/Install "create-react-app npm package globally/Install the `create-react-app` `npm` package globally/

Reply:
As this section is not needed, removed it